<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glowsuit Command Station</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vertical_layout">
     <item>
      <layout class="QHBoxLayout" name="horizontal_layout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="left_side_layout">
         <item>
          <widget class="QGroupBox" name="midi_file_group">
           <property name="toolTip">
            <string>Playback from a MIDI file and music file. If this is disable, uncheck the &quot;live MIDI&quot; box</string>
           </property>
           <property name="title">
            <string>Midi File Controls</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="2" column="0">
               <widget class="QPushButton" name="select_midi_file_button">
                <property name="text">
                 <string>Midi File</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="select_music_file_button">
                <property name="text">
                 <string>Music File</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="music_filename_label">
                <property name="text">
                 <string>no file select</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="midi_filename_label">
                <property name="text">
                 <string>no file selected</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QToolButton" name="stop_button">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Stops playback and resets to the start</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset theme="media-playback-stop">
                  <normaloff>.</normaloff>.</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="play_button">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset theme="media-playback-start">
                  <normaloff>.</normaloff>.</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="player_slider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="player_time_label">
                <property name="text">
                 <string>00:00/00:00</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>36</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="xbee_port_label">
             <property name="text">
              <string>XBee Serial Port</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="xbee_port_combobox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="controls_layout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="octave_label">
             <property name="text">
              <string>Oct +/-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="octave_spinbox">
             <property name="toolTip">
              <string>An offset for transposing MIDI from a file or from live MIDI</string>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <number>-2</number>
             </property>
             <property name="maximum">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="live_midi_controls">
           <property name="title">
            <string>Live Midi Controls</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="live_checkbox">
              <property name="toolTip">
               <string>Enables visualization of live MIDI. Disables the ability to play from a midi file</string>
              </property>
              <property name="text">
               <string>Live Midi?</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="midi_indicator_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../rc/command_station.qrc">
                <normaloff>:/icons/midi-off.png</normaloff>
                <normalon>:/icons/midi-on.png</normalon>:/icons/midi-off.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="viz_controls_layout">
           <item>
            <widget class="QPushButton" name="front_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>if red, don't show any of the EL wire visible only from in front</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
background-color: #e83535;
color: #000000;
}
QPushButton:pressed {
background-color: #ad1010;
color: #000000;
}
QPushButton:checked {
background-color: #58eb34;
color: #000000;
}
QPushButton:checked:pressed {
background-color: #34ba13;
color: #000000;
}</string>
             </property>
             <property name="text">
              <string>Front</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="back_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>if red, don't show any of the EL wire visible only from behind</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
background-color: #e83535;
color: #000000;
}
QPushButton:pressed {
background-color: #ad1010;
color: #000000;
}
QPushButton:checked {
background-color: #58eb34;
color: #000000;
}
QPushButton:checked:pressed {
background-color: #34ba13;
color: #000000;
}</string>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="scale_label">
             <property name="text">
              <string>Scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="scale_spinbox">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/command_station.qrc"/>
 </resources>
 <connections/>
</ui>
